### PR TITLE
feat: Add logs to feature config changes [WPB-1933]

### PR DESCRIPTION
**Pull request by atomrc**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-1933" title="WPB-1933" target="_blank">BugWPB-1933</a>  [MacOS?] Notification prompts received after logging in
  </td></table>
  <br />

### CODE REVIEW

File: src/script/page/FeatureConfigChangeNotifier.ts [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+      return undefined;
+    }
+    return {
+      htmlMessage: isFeatureEnabled
+        ? isEnforced
+          ? t('featureConfigChangeModalSelfDeletingMessagesDescriptionItemEnforced', {
+              timeout: formatDuration(newTimeout).text,
+            })
+          : t('featureConfigChangeModalSelfDeletingMessagesDescriptionItemEnabled')
+        : t('featureConfigChangeModalSelfDeletingMessagesDescriptionItemDisabled'),
```
Comment (przemvs) [inline src/script/page/FeatureConfigChangeNotifier.ts:100]: I'll move this to variable to avoid complex ternary operators.

**Review by przemvs — APPROVED** [review]
LGTM!

File: src/script/page/FeatureConfigChangeNotifier.ts (hunk 1 above)
Comment (atomrc) [reply src/script/page/FeatureConfigChangeNotifier.ts:100]: I think it's pretty readable like this (and backed up by tests now)

Comment (przemvs) [reply src/script/page/FeatureConfigChangeNotifier.ts:100]: I'll go this way:

```suggestion
const enforcedMessage = isEnforced
          ? t('featureConfigChangeModalSelfDeletingMessagesDescriptionItemEnforced', {
              timeout: formatDuration(newTimeout).text,
            })
          : t('featureConfigChangeModalSelfDeletingMessagesDescriptionItemEnabled');
          
      htmlMessage: isFeatureEnabled
        ? enforcedMessage
        : t('featureConfigChangeModalSelfDeletingMessagesDescriptionItemDisabled'),
```

**Pull request merged**